src/hooks/UseUpdateNotifications.ts

```diff
@@ -64,11 +64,18 @@ export function useUpdateNotifications(org: string, repo: string) {
       },
     },
   );
+  let errorCreationMessage = null;
```

Comment:
nit: could we just declare the variable and skip it's initialization to `null`?

Reply:
I prefer to initialize them to null since `null` checks are more common than `undefined` checks.